### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by aghassen**

### CODE REVIEW

**Comment by Houssem005** [comment]
LGTM

**Review by Houssem005 — COMMENTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -60,34 +66,37 @@ Submit the application to a Spark cluster:
...
 ```  
-spark-submit
+spark-submit 
 --class Quickstart 
 --master local[*] 
---packages io.delta:delta-core_2.12:2.0.0 
---jars lib/Lighthouse_2.12-0.1.0.jar 
-target/scala-2.12/clustering-metrics-example_2.12-0.1.jar
+--packages io.delta:delta-core_2.12:2.3.0,io.github.Databeans:lighthouse_2.12:0.1.0 
+target/scala-2.12/clustering-metrics-example_2.12-0.1.jarsbt
```
Comment (Houssem005) [inline README.md:84]: .jar at the end , not jarsbt

**Review by Houssem005 — COMMENTED** [review]
everything else is great

File: README.md (hunk 1 above)
Comment (aghassen) [reply README.md]: Done.

**Review by Houssem005 — APPROVED** [review]

**Pull request merged**